webapp/app/[locale]/_components/header.tsx

```diff
@@ -1,8 +1,8 @@
 import { CloseIcon } from 'components/icons/closeIcon'
 import { HamburgerIcon } from 'components/icons/hamburgerIcon'
+import { Link } from 'components/link'
```

Comment:
but doesn't this cause other issues related to i18n? I mean, does navigation work fine while mantaining the selected language?

Reply:
it works because this wrapper imports the Link from `next-intl` and not the link from `navigation/next`

Reply:
so it also preserves the locale (the locale is part of the `pathname`, and not a query string)